docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_to_postgresql.rst

```diff
@@ -1,18 +1,103 @@
+
+        stores:
+            validations_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: uncommitted/validations/
+
+3. **Update your configuration file to include a new store for Validations on PostgreSQL**
+
+    In our case, the name is set to ``expectations_postgres_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``DatabaseStoreBackend``, and ``credentials`` will be set to ``${db_creds}``, which references the corresponding key in the ``config_variables.yml`` file.
+
+    .. code-block:: yaml
+
+        expectations_store_name: validations_postgres_store
+
+        stores:
+            validations_postgres_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: DatabaseStoreBackend
+                    credentials: ${db_creds}
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+
+
+5. **Confirm that the new Validations store has been added by running** ``great_expectations store list``.
+
+    Notice the output contains two Validation stores: the original ``validations_store`` on the local filesystem and the ``validations_postgres_store`` we just configured.  This is ok, since Great Expectations will look for Validations in PostgreSQL as long as we set the ``validations_store_name`` variable to ``validations_postgres_store``. The config for ``validations_store`` can be removed if you would like.
+
+    .. code-block:: bash
+
+        great_expectations store list
+
+        - name: validations_store
+        class_name: ValidationsStore
+        store_backend:
+            class_name: TupleFilesystemStoreBackend
+            base_directory: uncommitted/validations/
+
+        - name: validations_postgres_store
+        class_name: ValidationsStore
+        store_backend:
+            class_name: DatabaseStoreBackend
+            credentials:
+                database: '<your_db_name>'
+                drivername: postgresql
+                host: '<your_host_name>'
+                password: ******
+                port: '<your_port>'
+                username: '<your_username>'
+
+
+
+6. **Confirm that the Validations store has been correctly configured.**
+
+    Run a :ref:`Checkpoint <tutorials__getting_started__set_up_your_first_checkpoint>` to store results in the new Validations store in PostgreSQL then visualize the results by re-building :ref:`Data Docs <tutorials__getting_started__set_up_data_docs>`.
+
+    Behind the scenes, Great Expectations will create a new table in your database called ``ge_validations_store``, and populate the fields with information from the Validation results.
```

Comment:
I feel like this will be useful for the user to know, but like the equivalent section in the Expectations Store document, I'm not sure if this is the right place to have this explanation.

Reply:
Hmm... I can't think of a better place for this. I suggest keeping it here.